.github/maintainers-little-helper.yaml

```diff
@@ -53,124 +53,142 @@ flake-tracker:
+        - cilium-master-k8s-1.25-kernel-4.19
+      Cilium-PR-K8s-1.17-kernel-4.19:
         correlate-with-stable-jobs:
-        - cilium-master-k8s-1.16-kernel-4.9
-        - cilium-master-k8s-1.17-kernel-4.9
-        - cilium-master-k8s-1.18-kernel-4.9
-        - cilium-master-k8s-1.19-kernel-4.9
-        - cilium-master-k8s-1.20-kernel-4.9
-        - cilium-master-k8s-1.21-kernel-4.9
-        - cilium-master-k8s-1.22-kernel-4.9
-        - cilium-master-k8s-1.23-kernel-4.9
-        - cilium-master-k8s-1.24-kernel-4.9
-      Cilium-PR-K8s-1.18-kernel-4.9:
+        - cilium-master-k8s-1.16-kernel-4.19
+        - cilium-master-k8s-1.17-kernel-4.19
+        - cilium-master-k8s-1.18-kernel-4.19
+        - cilium-master-k8s-1.19-kernel-4.19
+        - cilium-master-k8s-1.20-kernel-4.19
+        - cilium-master-k8s-1.21-kernel-4.19
+        - cilium-master-k8s-1.22-kernel-4.19
+        - cilium-master-k8s-1.23-kernel-4.19
+        - cilium-master-k8s-1.24-kernel-4.19
+        - cilium-master-k8s-1.25-kernel-4.19
+      Cilium-PR-K8s-1.18-kernel-4.19:
         correlate-with-stable-jobs:
-        - cilium-master-k8s-1.16-kernel-4.9
-        - cilium-master-k8s-1.17-kernel-4.9
-        - cilium-master-k8s-1.18-kernel-4.9
-        - cilium-master-k8s-1.19-kernel-4.9
-        - cilium-master-k8s-1.20-kernel-4.9
-        - cilium-master-k8s-1.21-kernel-4.9
-        - cilium-master-k8s-1.22-kernel-4.9
-        - cilium-master-k8s-1.23-kernel-4.9
-        - cilium-master-k8s-1.24-kernel-4.9
-      Cilium-PR-K8s-1.19-kernel-4.9:
+        - cilium-master-k8s-1.16-kernel-4.19
+        - cilium-master-k8s-1.17-kernel-4.19
+        - cilium-master-k8s-1.18-kernel-4.19
+        - cilium-master-k8s-1.19-kernel-4.19
+        - cilium-master-k8s-1.20-kernel-4.19
+        - cilium-master-k8s-1.21-kernel-4.19
+        - cilium-master-k8s-1.22-kernel-4.19
+        - cilium-master-k8s-1.23-kernel-4.19
+        - cilium-master-k8s-1.24-kernel-4.19
+        - cilium-master-k8s-1.25-kernel-4.19
+      Cilium-PR-K8s-1.19-kernel-4.19:
         correlate-with-stable-jobs:
-        - cilium-master-k8s-1.16-kernel-4.9
-        - cilium-master-k8s-1.17-kernel-4.9
-        - cilium-master-k8s-1.18-kernel-4.9
-        - cilium-master-k8s-1.19-kernel-4.9
-        - cilium-master-k8s-1.20-kernel-4.9
-        - cilium-master-k8s-1.21-kernel-4.9
-        - cilium-master-k8s-1.22-kernel-4.9
-        - cilium-master-k8s-1.23-kernel-4.9
-        - cilium-master-k8s-1.24-kernel-4.9
-      Cilium-PR-K8s-1.20-kernel-4.9:
+        - cilium-master-k8s-1.16-kernel-4.19
+        - cilium-master-k8s-1.17-kernel-4.19
+        - cilium-master-k8s-1.18-kernel-4.19
+        - cilium-master-k8s-1.19-kernel-4.19
+        - cilium-master-k8s-1.20-kernel-4.19
+        - cilium-master-k8s-1.21-kernel-4.19
+        - cilium-master-k8s-1.22-kernel-4.19
+        - cilium-master-k8s-1.23-kernel-4.19
+        - cilium-master-k8s-1.24-kernel-4.19
+        - cilium-master-k8s-1.25-kernel-4.19
```

Comment:
It seems these lists all have a new entry now for 1.25. Why?
I think I'm missing how those `correlate-with-stable-jobs` are used.

Reply:
MLH correlation uses the matrix defined in config to correlate PR jobs with scheduled jobs on `master`.

For example, related to the lines outlined above: `Cilium-PR-K8s-1.19-kernel-4.19` (i.e. `
k8s-1.19-kernel-4.19 (test-1.19-4.19)`) will be correlated with all 4.19 `master` jobs.

The reason 1.25 is a "new" entry is because we already had 4.19 testing on 1.25, so now that 1.16-1.24 also use 4.19 it makes sense to correlate against it as well since it's the same kernel.

Reply:
Got it. Thanks for the explanation!